TSOClient/tso.client/UI/Controls/UIVMPersonButton.cs

```diff
@@ -66,7 +67,8 @@ public override void Update(UpdateState state)
                 UpdateAvatarState(perm);
             }
             RMB = state.MouseState.RightButton == ButtonState.Pressed;
-            if (RMB && Hovered) CenterPerson(this);
+            RMBscroll = GameFacade.Cursor.CurrentCursor != Common.Rendering.Framework.CursorType.Normal;
```

Comment:
That's kind of a hack, it's probably just better setting a local variable `rmb` to the current click state then checking `RMB && !rmb` to see if the right mouse button was clicked (wasn't down last frame, is down now) and then checking if the mouse position is on our bounds. Set private `RMB = rmb` afterwards so you store the last state we saw.